### PR TITLE
Add tap-shopify as a connector

### PR DESCRIPTION
**Pull request by alastairstuart**
## Problem

Adds support for extracting Shopify e-commerce data via tap-shopify

## Proposed changes

Adds a connector and surrounding documentation for using tap-spotify

## Types of changes

What types of changes does your code introduce to PipelineWise?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [ ] CI checks pass with my changes
- [ ] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [ ] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [ ] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: docs/connectors/taps/shopify.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
+
+.. _tap-shopify:
+
+Tap Shopify
+-----------
+
+Configure access to your Shopify store
+''''''''''''''''''''''''''''''''''''''
+
```
Comment (alastairstuart) [inline docs/connectors/taps/shopify.rst:9]: These instructions are from https://meltano.com/plugins/extractors/shopify.html

**Comment by Samira-El** [comment]
Hey Al, thanks for the PR, this will be a great addition to the PipelineWise family!

There are few more places to update, please add the tap to:

* [Docs landing page](https://github.com/transferwise/pipelinewise/blob/master/docs/index.rst)
* The table listing the connectors in the [Readme file](https://github.com/transferwise/pipelinewise/blob/master/README.md#connectors)
* Extra connectors in the installation [script](https://github.com/transferwise/pipelinewise/blob/e0a8ee15c05f019916f5400158b81de72cf33dc8/install.sh#L182)
* [Installation guide](https://github.com/transferwise/pipelinewise/blob/master/docs/installation_guide/installation.rst)
* [YAML config](https://github.com/transferwise/pipelinewise/blob/master/docs/user_guide/yaml_config.rst)

**Comment by alastairstuart** [comment]
@Samira-El are those documentation changes OK?

**Comment by Samira-El** [comment]
Hey Al, looks good.

**Review by Samira-El — APPROVED** [review]

**Pull request merged**